Dockerfile

```diff
@@ -1,6 +1,6 @@
 #
 # --- Base Node Image ---
-FROM node:8-alpine AS base
+FROM node:12-alpine AS base
```

Comment:
This one and the one below should probably use the node tag, `lts-alpine` instead of `12` directly.

Reply:
It seems, `12` builds, but `lts` doesn't. It may be something in the npm packages as this occurs with lts: https://github.com/netreconlab/parse-dashboard/runs/3545759058?check_suite_focus=true#step:7:378

Reply:
ci tests don't pass for node 16 because node-sass (also in the logs you linked) needs to be bumped. maybe that's why? not sure which node version lts-alpine uses.

Reply:
> ci tests don't pass for node 16 because node-sass (also in the logs you linked) needs to be bumped it seems. maybe that's why?

Makes sense, should it be bumped in this PR?

Reply:
I tried, it seems to requires some code refactoring because this is a chain reaction where we need to bump webpack to 5 as well. Dashboard is currently not officially Node 16 tested, that will be an undertaking of its own.